### DISCUSSION TITLE
Light card not using all the free space?

### DISCUSSION THREAD

**Discussion by Alfamonk** [piitaya/lovelace-mushroom#694] [General]
Hello!
Is there a way to force the color/brightness/temperature slider so that it uses the whole free space (next to Name)?
It's really bad and small, specially on the smartphones, and there is way too much wasted space on computer. I know I could set it vertically, but then it takes too much space.
Maybe is there some kind of option to make the slider bigger? I've tried "Fill container" but got no luck...
Thanks in advance!

![IMG_6116](https://user-images.githubusercontent.com/87588396/185752145-4c476b5c-85ff-4f31-bab8-73f93a873126.jpeg)
![2022-08-20_174147](https://user-images.githubusercontent.com/87588396/185752147-7e4c3696-2a1f-4d4e-98c1-c7a41e68488a.png)

**Comment by piitaya** [comment]
Controls use 50% of the space. Entity name can be longer so it's 50% to keep a design consistency.

**Reply by Alfamonk** [reply]
> Controls use 50% of the space. Entity name can be longer so it's 50% to keep a design consistency. 

Can't we do a custom thing, to choose 75% and 25%?

**Reply by piitaya** [reply]
Currently it's not possible but this can be a theme variable.

**Comment by antonio1475** [comment]
@Alfamonk I solve this with:

```
            card_mod:
              style: |
                mushroom-state-item {margin-right: -23%;}
```
 You’ll need to adjust the % to your devices/liking/light name length.

![230BC1DF-57C3-43B7-9356-1A45E4FAA93B](https://user-images.githubusercontent.com/63130724/194780430-af284cdd-342c-4646-a4a2-3e7d4e73953d.png)
